package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@whereby.com/browser-sdk",
-    "version": "2.0.0-alpha27",
+    "version": "2.0.0-alpha28",
```

Comment:
ahh I merged alpha26 this morning, hoping we can go live together with your change. But in the meantime there was another merge (alpha27) and it was not deployed either.

So if you merge this, then the next tag is alpha28, and it will look like there are 2 tags missing :/ It's not a big problem IMO, but looks bad.

A fix would be to actually downgrade here to `alpha26` (kind of reverting the other pr's version bump) and deploy these 3 things together in `alpha26`. WDYT?

Reply:
sure, I can downgrade to `alpha26`, no problem